app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @id = current_user.id
+    @comment = Comment.new
+  end
+
+  def create
+    post = Post.find(params[:id])
+    comment = Comment.new(user: current_user, post:, text: post_params[:text])
+    if comment.save
+      flash[:notice] = 'Comment created successfully'
+    else
+      flash[:error] = 'Error'
+    end
+    redirect_to user_post_comments_path
+  end
+
+  private
+
+  def post_params
```

Comment:
[ OPTIONAL]
- I would suggest giving this method a relatable name like `comment_params`